lib/src/main/java/io/github/taz/java/instagram/api/requests/IgGetRequest.java

```diff
@@ -18,6 +21,7 @@ public IgGetRequest(Class<T> responseType, String path, Map<String, Object> quer
 
 	@Override
 	public HttpRequest formRequest(IgClient client) {
+		logger.info("Sending POST request to {}", this.getUrl());
```

Comment:
this just makes a post request, it doesn't send it. Also i don't feel like we should have logging here

Reply:
I think we should log it though, as it is a networking operation, one which other classes are based on.

Reply:
it is not networking operation, it is just creating an object which will be sent later

Reply:
oh right, I realize that now, will remove it.